components/cli/pkg/util/utils.go

```diff
@@ -25,7 +25,7 @@ import (
 	"crypto/tls"
 	"errors"
 	"fmt"
-	"github.com/manifoldco/promptui"
+	"github.com/miraj/sdk/components/cli/pkg/util"
```

Comment:
Shouldn't this be wso2-celley ?

Reply:
This is an unnecessary import by the IDE. removed it.